lms/envs/common.py

```diff
@@ -340,9 +340,6 @@
     # Enable OpenBadge support. See the BADGR_* settings later in this file.
     'ENABLE_OPENBADGES': False,
 
-    # The block types to disable need to be specified in "x block disable config" in django admin.
-    'ENABLE_DISABLING_XBLOCK_TYPES': True,
```

Comment:
Eric faced something similar with another setting he moved to a configuration model (related to bulk email). He talked to Ned, and Ned was fine with not providing a migration file and just telling the open source community what to do. I had also talked to Scott about whether or not to do a migration file, and we came to the decision not to do a migration file.

The fact that only a small number of items are currently likely to be in the database (I wouldn't expect that open source deployments are using this setting at all) also lends support to not providing a migration script.